cluster/userdata-master.yaml

```diff
@@ -1028,6 +1028,59 @@ write_files:
             hostNetwork: true
             hostPID: true
 
+  - path: /srv/kubernetes/manifests/daemonsets/skipper-ingress.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: DaemonSet
+      metadata:
+        name: skipper-ingress
+        namespace: kube-system
+        labels:
+          application: skipper-ingress
+          version: prerelease-ingress-1
+          component: ingress
+      spec:
+        selector:
+          matchLabels:
+            application: skipper-ingress
+        template:
+          metadata:
+            name: skipper-ingress
+            labels:
+              application: skipper-ingress
+              version: prerelease-ingress-1
+              component: ingress
+            annotations:
+              scheduler.alpha.kubernetes.io/critical-pod: ''
+              scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
+          spec:
+            containers:
+            - name: skipper-ingress
+              image: registry.opensource.zalan.do/teapot/skipper:prerelease-ingress-1
+              ports:
+              - name: ingress-port
+                containerPort: 9999
+                hostPort: 9999
+              command: ["skipper", "-application-log-level", "DEBUG", "-kubernetes-url", "http://localhost:8001", "-address", ":9999"]
+              resources:
+                limits:
+                  cpu: 200m
+                  memory: 200Mi
+                requests:
+                  cpu: 25m
+                  memory: 25Mi
+            - name: kubectl
+              image: registry.opensource.zalan.do/teapot/hyperkube:v1.5.0-beta.2_custom.0
```

Comment:
It would make sense to use the same hyperkube image here as we do in all other places.

This is currently: `registry.opensource.zalan.do/teapot/hyperkube:v1.4.7_zalando.0`